README.md

```diff
@@ -73,7 +73,8 @@ To deploy an existing model on your machine, you can either 1) specify the name
 ### Option 1: use a model from the Viam registry
 
 You can download an ML model from Viam's registry. Head to
-https://app.viam.com/registry?type=ML+Model to see the available models. Once you've added this
+https://app.viam.com/registry?type=ML+Model to see the available models. Choose a model that uses either
+the TensorFlow or PyTorch model framework; this service does not support TensorFlow Lite. Once you've added this
```

Comment:
```suggestion
https://app.viam.com/registry?type=ML+Model to see the available models. Choose a model that uses a [supported model framework](https://docs.nvidia.com/deeplearning/triton-inference-server/user-guide/docs/backend/docs/backend_platform_support_matrix.html). Once you've added this
```
While I would prefer to link higher up in the readme, I'm concerned that that might not translate well to the Viam app, so I opted to play it safe and link to the third-party doc instead.